File: python/section2/q12.py
from q10 import load_pure_hightmp

if __name__ == '__main__':
    cols = load_pure_hightmp()
    for i in 1, 2:
        with open('col%d.txt' % i, 'w') as f:
            for col in cols:
                print(col.split("\t")[i-1], file=f)

File: scala/section2/src/main/scala/q12.scala
import java.io.File
import java.io.PrintWriter

import scala.io.Source

/**
  * Created by nomotoeriko on 2017/06/08.
  */
object q12 {
  def main(args: Array[String]): Unit = {
    val sentence = q10.load_sentence()
    val col1 = new PrintWriter(new File("col1.txt"))
    val col2 = new PrintWriter(new File("col2.txt"))

    sentence.foreach(x => col1.write(x.split("\t").head+"\n"))
    sentence.foreach(x => col2.write(x.split("\t").tail.head+"\n"))
    col1.close()
    col2.close()
  }
}
